database/seeders/DatabaseSeeder.php

```diff
@@ -13,6 +14,7 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
+        Category::factory()->count(10)->create();
```

Comment:
What do you think about this idea?
Let's create 10 categories that you selected in this seeder instead of random ones.

You can do it like this:
```php
Category::factory()->create([
    'title' => 'food',
    'icon' => 'bag',
]);
```